packages/node/test/unit/ConfigWizard.test.ts

```diff
@@ -78,10 +78,29 @@ const IMPORTED_PRIVATE_KEY =
 
 const OPERATOR_ADDRESS = '0x54d68882d5329397928787ec496da3ba8e45c48c'
 
+const parseStoragePath = (summary: string): string | undefined => {
+    const match = summary.match(/streamr-node ([^\s]w+)/)
+    if (match !== null) {
+        return match[1]
+    } else {
+        return undefined
+    }
```

Comment:
`match` can only be a `null` or a "match".

I'd consider either

```ts
// 1
return match?.[1] ?? undefined
```

or 

```ts
// 2
return match ? match[1] : undefined
```

or at least

```ts
// 3
if (match !== null) {
    return match[1]
}

return undefined
```

Reply:
Applied the ternary with explicit `null` expression: `return (match !== null) ? match[1] : undefined`